src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs

```diff
@@ -21,6 +21,9 @@ public class NameValueHeaderValue
 
         private StringSegment _name;
         private StringSegment _value;
+        private bool _quotedValue;
```

Comment:
Why save this as a field? `HeaderUtilities.IsQuoted()` doesn't look that expensive.